Assets/Scripts/Utilities/CCDebug.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+public enum LogLevel {Error, Warning, Info, Verbose }
+public enum LogMessageCategory {All, Rendering, Networking, VR, UI, Interaction, Event}
+
+public static class CCDebug
+{
+    public static LogLevel[] CurrentLevels = new []
+    {
+        LogLevel.Error,
+        LogLevel.Warning,
+        LogLevel.Info,
+        LogLevel.Verbose
+    };
+
+    public static LogMessageCategory[] Categories = new[]
+    {
+        LogMessageCategory.All,
+        LogMessageCategory.Event,
+        LogMessageCategory.Interaction,
+        LogMessageCategory.Networking,
+        LogMessageCategory.Rendering,
+        LogMessageCategory.UI,
+        LogMessageCategory.VR
+    };
+    
+    public static void Log(object logMessage, LogLevel level, LogMessageCategory category)
+    {
+        if (CurrentLevels.Contains(level) && Categories.Contains(category))
+        {
+            switch (level)

```

Comment:
I can see a case for using a numeric representation for level and doing <> compare on it, but this is fine too.

Reply:
Agree with feedback - this was an experiment, but I do prefer the compare you suggested.